src/main/java/Test.java

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
1. you mean to publish to maven ? Does it really make sense to have that in the readme ?
2. ok